src-built-in/components/toolbar/stores/toolbarStore.js

```diff
@@ -201,7 +207,9 @@ class _ToolbarStore {
 	 */
 	bringToolbarToFront(focus) {
 		var self = this;
-		finsembleWindow.bringToFront(null, () => {
+		finsembleWindow.bringToFront(null, (err) => {
+			if (err) { FSBL.Clients.Logger.error(`finsembleWindow.bringToFront failed, error:`, err); }
```

Comment:
Already using template literal, no point of using them if going to pass multiple arguments. I would remove the coma and put `err` within the ` `

Reply:
If err is an object. the string will say "[object object]" instead of logging the actual error.